docs/guides/stay-up-to-date/terraform/how-to-update-to-aws-provider-v4/index.md

```diff
@@ -0,0 +1,34 @@
+# Upgrade to AWS Provider v4
+
+This guide will walk you through how to update the [Gruntwork Reference
+Architecture](https://gruntwork.io/reference-architecture/) and any code that depends on the
+[Gruntwork Infrastructure as Code Library](https://gruntwork.io/infrastructure-as-code-library/) to version 4.x of the
+Terraform AWS provider. Following the release of v4.0.0, new features and bug fixes will only be available on version
+4.x, but it also has a number of backward incompatibilities that have to be incorporated into your codebase.
+
+## What you’ll learn in this guide
+
+This guide consists of three main sections:
```

Comment:
There are only two sections below, though?

Reply:
That's strange. We already updated that in [84dac5c](https://github.com/gruntwork-io/docs/pull/491/commits/84dac5c6fbb7539a2268c73bff0c5f00b19d9353) 👀